### PR TITLE
PCR-520,542,536,535,515

### PR DESCRIPTION
**Pull request by kshitijzutshi**
### PCR-520,542,536,535,515
image

### CODE REVIEW

File: web/rainmaker/dev-packages/egov-rented-properties-dev/src/ui-config/screens/specs/rented-properties/search-preview.js [hunk 1]
```diff
@@ -67,7 +67,13 @@ export const searchResults = async (action, state, dispatch, transitNumber) => {
   let payload = await getSearchResults(queryObject);
   if(payload) {
     let properties = payload.Properties;
-    // properties = properties[0].owners.filter(itemdat => itemdat.permanent === true)
+    let i;
+    for (i = 0; i < properties[0].owners.length; i++) {
```
Comment (tejaswi-transerve) [inline web/rainmaker/dev-packages/egov-rented-properties-dev/src/ui-config/screens/specs/rented-properties/search-preview.js:71]: change to array map function

Comment (kshitijzutshi) [reply web/rainmaker/dev-packages/egov-rented-properties-dev/src/ui-config/screens/specs/rented-properties/search-preview.js:71]: Will update PR

**Comment by kshitijzutshi** [comment]
will update PR with change

**Comment by kshitijzutshi** [comment]
Changes done in map fn

**Comment by kshitijzutshi** [comment]
changing the map fn for possession date alteration

**Comment by kshitijzutshi** [comment]
possession date owners array map change done

**Pull request merged**